### DISCUSSION TITLE
Help Required on clustered High Availability Harbor Registry in VM

### DISCUSSION THREAD

**Discussion by mail2sandeepd** [goharbor/harbor#16672] [Q&A] [answered]
What can we help you?

We have single node Harbor Registry in VM, We need a clustered High Availability Harbor Registry in VM, Can someone suggest any document or tips, How to configure Harbor Registry Cluster on VM as Active/Active or Active/Passive.

Any Tips, Document, Suggestion will be much appreciated.

Thanks.

**Comment by Vad1mo** [comment, marked as answer]
As a starting point take a look at https://goharbor.io/docs/edge/install-config/harbor-ha-helm/

**Reply by mail2sandeepd** [reply]
Hi @Vad1mo, 
we don't have kubernetes environment, hence we need HA to be configured on Virtual servers.

please guide on this .

**Reply by Vad1mo** [reply]
There is no specific documentation on how to do it outside k8s. There would be just too many variants. 

IMHO you can still use https://goharbor.io/docs/edge/install-config/harbor-ha-helm/ as a reference, as the overall concept decribed generally apply
